### PR TITLE
Issue #56 and #51 addressed

### PR DESCRIPTION
**Pull request by Aniq55**
Issue #51 
Added screenshot to the README
Issue #56 
Prints error message when the pics folder is empty

### CODE REVIEW

File: scheduler.py [hunk 1]
```diff
@@ -15,11 +15,16 @@ def __init__(self, offline):
...
-        self.change_random()
-        self.setStartTime(time.time())
-        self.changeCycle()
+        
+        if os.listdir(directory) != []:
+            self.change_random()
+            self.setStartTime(time.time())
+            self.changeCycle()
+        else:
+            print("No downloaded images. Try again in online mode.")
```
Comment (abhsag24) [inline scheduler.py]: awesome! :D

File: README.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ https://gitter.im/NIT-dgp/General
 Screenshot
 ==========
  
- ![Alt text](UIP/UIP_screenshot.png ?raw=true "Screenshot (Ubuntu)")
+ ![alt tag](https://github.com/Aniq55/UIP/blob/master/UIP_screenshot.png?raw=true )
```
Comment (abhsag24) [inline README.md]: why was this required at all?  didn't it work with the last commit?

**Comment by abhsag24** [comment]
@Aniq55 
1. so commit 3099c53 and 96d5659 are the same right? why are there 2 commits here? 
2. commit f7af12d is perfect and excellent. 
3. commit c86de0b also needs to be merged with 3099c53 or 96d5659 as they are the same change       also the screenshot needs to go into a new examples folder and also the link in 3099c53 is wrong, it's a link to your repo, i think you can easily escape this problem as links are required to be relative, no need for a direct link.

If you any problem addressing any/ all of the issues please contact me on gitter or just reply here.

**Comment by Aniq55** [comment]
Okay. I've understood.

**Comment by abhsag24** [comment]
So this might seem a daunting task to correct all those commits feel free
to contact me whenever needed

**Comment by abhsag24** [comment]
@Aniq55 the change to README.md is still missing? :(

**Comment by abhsag24** [comment]
@Aniq55 awesome PR! no changes need to be done to ther code,  just one technical thing i need you to do before this gets merged, can you merge 0298ef6 and 	240e4c1 together? they are basically the same logical change?

File: scheduler.py (hunk 1 above)
Comment (Aniq55) [inline scheduler.py]: thanks. (:

**Comment by Aniq55** [comment]
@abhsag24 I dont know how to merge commits, I'll check online though.

**Comment by Aniq55** [comment]
@abhsag24 I've squashed the two commits you asked me to.

**Comment by abhsag24** [comment]
@Aniq55 why do you have @mohitshaw's comment? remove that?

**Comment by Aniq55** [comment]
I dont know how that came here.

**Comment by abhsag24** [comment]
@Aniq55 rebase and remove that commit ping me on gitter if you need help rebasing, also i dont see the change in the README? :(
